environments/softlayer/inventory.ini

```diff
@@ -96,13 +100,15 @@ formplayer0
 
 [redis:children]
 db0
+db1
```

Comment:
was just commenting here https://github.com/dimagi/commcare-cloud/pull/1710#pullrequestreview-119000245

It might be better to remove db0 completely and just deploy from this branch until ready to flip?